templates/documents/collection_details.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'documents/shared/details_base.html' %}
+
+{% block detail_content %}
+    <div class="col-span-3">
+        <input x-cloak x-show="editing" name="name" class="input input-bordered" value="{{ object.name }}"></input>
+        <h1 x-show="!editing" class="pg-title">{{ object.name }}</h1>
+
+        <dl class="grid grid-cols-2 gap-1">
+            <div class="py-3 col-span-1">
+                <dt class="text-sm font-medium leading-6">Uploaded at</dt>
```

Comment:
`leading` changes the line height which is usually used for prose. For vertical spacing you should use margin or padding. If you need alignment then `align-x`